src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const obj = state;
```

Comment:
do copy, now obj === state

Reply:
`const obj = { ...state };`
if i return a copy, tests fail 11/13
![image](https://user-images.githubusercontent.com/114029531/224819632-0d53ae60-c74c-4e79-8f4c-0c3fda3900bb.png)
What i have to do?